platforms/Linux/DEB/Ubuntu/focal/debian/control.in

```diff
@@ -0,0 +1,46 @@
+Source: swiftlang
```

Comment:
how is `platforms/Linux/DEB/Ubuntu/focal/debian/control` different from `platforms/Linux/DEB/Ubuntu/focal/debian/control.in`? the _look_ the same?

Reply:
They are identical before the package build starts; the reason is that the tools need a valid, reasonable `control` file in place when the build starts. Templating the `control` file is not the most common thing to do and needs precautions like this to actually work :)

Reply:
`control` is generated from `control.in` in the `clean` rule, but needs to exist prior to that, with valid content. That's why they are identical here.

Reply:
interesting. what is the expected maintenance workflow? ieif we need to make updates, they go into `control.in` and `control` can stay behind? or we always update both?

Reply:
yea I think that would be clearer for future maintainers